### ISSUE TITLE
Add a11y test to make sure stories are loading

### ISSUE THREAD

**Issue by interactivellama** [salesforce/design-system-react#2226]
Currently, there isn't a check to make Storybook stories are loading. This means that all tests pass--which is unfortunately what was happening then the original pull request was made. No tests were actually being run, so all the tests passed!

**Comment by stroudn1** [comment]
Hi @interactivellama ! Can I have some more info on what's needed? I ran a Lighthouse test and it looks like accessibility is fine but performance is at a 19? But I can't find anything regarding Storybook stories. Not sure if I'm understanding correctly. Thanks!

**Comment by interactivellama** [comment]
We use axe-puppeteer in the CI tests to run aXe on all the stories in storybook. Currently if the page never loads in the headless Chrome, then the tests still pass.

In our codebase, aXe passes if no HTML is loaded into headless Chrome. We need to make our CI tests fail if no DOM is loaded. 

This task would add a test to make sure that a DOM is loaded that can be tested to make sure there are no false positives.

I'm thinking that just adding a test here that checks for `#root` might work. Right now if `#root` doesn't exist then there's no test fail and the a11y test suite "passes" since there are "no a11y errors" when there is web page.
https://github.com/salesforce/design-system-react/blob/master/tests/axe-puppeteer-matcher.js#L58
